src/plans/stepsize.jl

```diff
@@ -756,6 +757,60 @@ function show(io::IO, a::NonmonotoneLinesearch)
 end
 get_message(a::NonmonotoneLinesearch) = a.message
 
+@doc raw"""
+    PolyakStepsize <: Linesearch
+
+
+Compute a step size
+
+````math
+α_i = \frac{f(p^{(k)} - f_{\text{best}}+\gamma_k}{\lVert \operatorname{grad} f(p^{(k)})} \rVert},
+````
+
+where ``f_{\text{best}}`` is the (up to now) best cost function seen until now,
+and ``\gamma_i`` is a sequence of numbers that is square summable, but not summable.
```

Comment:
I think you should mention `grad f` here, and also note that this stepsize also works for subgradients.

Reply:
for now I just mentioned the two things stored in the struct, but you are right we need a second variant for sub gradients as well of course.